sdk/template/azure-template/azure/template/_version.py

```diff
@@ -1,2 +1,2 @@
 # matches SEMVER
-VERSION = "0.0.13b1"
\ No newline at end of file
+VERSION = "0.0.13b2"
```

Comment:
Looks like https://github.com/Azure/azure-sdk-for-python/tree/azure-template_0.0.18b1 is the latest tag we have pushed. So please bump this to 0.0.18b2